### PR TITLE
[VWROOM-276] Add some info about `dequeue_mode`

### PR DESCRIPTION
**Pull request by todor-macrometa**
Jira: https://macrometa.atlassian.net/browse/VWROOM-276

This field have been added back in VWRS version 1.26.0. It'll be good if we add some information about it in the docs.

It's a tiny change, and the PR can be merged as soon as it's approved.

### CODE REVIEW

File: docs/photoniq/vwrs/configure-domain.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ The following optional fields define the behavior of the waiting room:
 
 - **queue_type**: Defines how requests should be removed from the waiting room. The three possible queue types are `fifo`, `random`, and `lottery`. If this is not set, then the default queue is `fifo`.
 - **queue_mode**: You can configure the waiting room to be enabled dynamically. When set to `auto`, the waiting room is enabled after reaching the defined `rate_limit` for a specific `metric_interval`. If set to `manual`, then the waiting room is always enabled.
+- **dequeue_mode**: When the value is `on`, users in the waiting room will be granted access to the origin at a rate given by `rate_limit`. When the value is `off`, users will be kept in the waiting room indefinitely.
```
Comment (diana-macrometa) [inline docs/photoniq/vwrs/configure-domain.md]:
```suggestion
- **dequeue_mode**: When the value is `on`, users in the waiting room are granted access to the origin at a rate given by `rate_limit`. When the value is `off`, users remain in the waiting room indefinitely.
```

**Review by diana-macrometa — APPROVED** [review]
Minor edit, then g2g.

File: docs/photoniq/vwrs/configure-domain.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ The following optional fields define the behavior of the waiting room:
...
 - **origin_key**: Every domain is associated with an origin that was created with the Create Origin REST API, POST `/api/vwr/v1/origins/{origin_id}`.
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/configure-domain.md]: Did we not remove the origins api?  Should we remove this field?

**Review by joubert-macrometa — APPROVED** [review]
@todor-macrometa one comment to look at about the origin API.

I am approving.  But resolve my comment.

**Pull request merged**